jenkins-slaves/jenkins-slave-golang/Dockerfile

```diff
@@ -13,6 +13,8 @@ RUN mkdir -p /usr/src/go/src/redhat && \
     chown -R 1001 /usr/local/go && \
     rm -f /usr/go${GO_VERSION}.linux-amd64.tar.gz
 
+RUN go get -u github.com/golang/dep/cmd/dep
```

Comment:
I'm guessing this one got in here by mistake and is already part of #134

Reply:
@pabrahamsson that's definitely a mistake. too much going on at once :)